Dockerfile

```diff
@@ -145,7 +145,7 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
     curl -ksSL https://github.com/nexB/scancode-toolkit/archive/v$SCANCODE_VERSION.tar.gz | \
         tar -zxC /usr/local && \
         # Trigger configuration for end-users.
-        /usr/local/scancode-toolkit-$SCANCODE_VERSION/scancode --version && \
+        PYTHON_EXE=/usr/bin/python3 /usr/local/scancode-toolkit-$SCANCODE_VERSION/scancode --version && \
```

Comment:
Would it make sense to leave a code comment saying that this actually configures ScanCode to use Python 3 ?

Reply:
I'd say that's pretty much obvious, due to a lack of alternatives what else it could do, or?

Reply:
But I'll amend the existing comment anyway.